git/.gitconfig

```diff
@@ -24,7 +24,7 @@
 [filter "trimWhitespace"]
 	clean = git-stripspace
 [push]
-	default = matching
+	default = simple
```

Comment:
This is great. I have messed up couple of time with matching option. now it will be very safe.